contents/handbook/growth/marketing/blog.md

```diff
@@ -4,11 +4,11 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Content is a key pillar of our marketing strategy. As our products are quite technical in nature, we believe that frequent and regular content output helps improve acquisition and ongoing product engagement. The [blog](/blog) is where we publish interesting product content, share feature updates, and tell PostHog stories.
+Content is the main pillar of our marketing strategy. As our target users are engineers, we believe that frequent and regular content output helps us to drive word of mouth, leading to signups. The [blog](/blog) is where we publish interesting product content, share feature updates, and tell PostHog stories.
 
 ## Content & SEO strategy
 
-Content SEO is a long-term strategy. We are focused on generating a sustained output of high quality content that is genuinely valuable to users.
+Content & SEO is a long-term strategy. We are focused on generating a sustained output of high quality content that is genuinely valuable to users. Rather than scale up content output for the sake of it, our strategy is to go _deeper_ and creating better content as we grow, not simply pushing out more articles for its own sake. For this reason, we never set ourselves specific content output targets. 
```

Comment:
maybe add somehow "We win on depth _not_ breadth of content." ? The point being that we won't beat our competitors on their terms, since we're smaller.

Reply:
Yeah agreed - I'll add that in.